garak/harnesses/base.py

```diff
@@ -104,24 +104,19 @@ def run(self, model, probes, detectors, evaluator, announce_probe=True) -> None:
                 continue
 
             attempt_results = probe.probe(model)
-            assert isinstance(attempt_results, list)
+            assert isinstance(
+                attempt_results, list
```

Comment:
Should this test for `Iterable` vs `list`?

Reply:
Good catch. Should check for ordered iterables, i.e. not `set` which can break alignment (I think). Updated to use same test as in `tests.detectors.test_detectors.test_detector_detect`.